arrow-avro/src/writer/encoder.rs

```diff
@@ -812,6 +848,10 @@ enum Encoder<'a> {
     Int(IntEncoder<'a, Int32Type>),
     Long(LongEncoder<'a, Int64Type>),
     Timestamp(LongEncoder<'a, TimestampMicrosecondType>),
+    DurationSec(LongEncoder<'a, DurationSecondType>),
+    DurationMs(LongEncoder<'a, DurationMillisecondType>),
+    DurationUs(LongEncoder<'a, DurationMicrosecondType>),
+    DurationNs(LongEncoder<'a, DurationNanosecondType>),
```

Comment:
Nit: let's stick to the naming convention we're using else where imo

```suggestion
    DurationSeconds(LongEncoder<'a, DurationSecondType>),
    DurationMillis(LongEncoder<'a, DurationMillisecondType>),
    DurationMicros(LongEncoder<'a, DurationMicrosecondType>),
    DurationNanos(LongEncoder<'a, DurationNanosecondType>),
```

As an aside the fact Avro refers to it's calendar time type as `Duration` just makes this confusing. lol